math/s2geometry/Portfile

```diff
@@ -36,12 +36,13 @@ platform powerpc {
 }
 
 # C++ standard should match that of abseil port:
-compiler.cxx_standard   2014
+compiler.cxx_standard   2017
 
 configure.args-append \
                     -DBUILD_SHARED_LIBS=ON \
                     -DBUILD_EXAMPLES=OFF \
-                    -DBUILD_TESTS=OFF
+                    -DBUILD_TESTS=OFF \
+                    -DCMAKE_CXX_STANDARD=17
```

Comment:
shouldn't this use the option `cmake.set_cxx_standard` from the PortGroup instead of adding it manually?

Reply:
I wasn't aware of that option and just copied what I saw in the change of abseil itself. 
I think you are right. Should I then place this directly below the calls to `compiler.cxx_standard`?